config/src/config/storage_config.rs

```diff
@@ -9,6 +9,7 @@ use std::{net::SocketAddr, path::PathBuf};
 #[serde(default, deny_unknown_fields)]
 pub struct StorageConfig {
     pub address: SocketAddr,
+    pub simple_address: SocketAddr,
```

Comment:
Is the plan to remove `address` and rename `simple_address` to `address`?

Reply:
Yes, after the clean up is totally done. Should I add a comment here?